index.js

```diff
@@ -20,14 +20,6 @@ module.exports = function (a, b) {
 	var aLen = a.length;
 	var bLen = b.length;
 
-	if (aLen === 0) {
-		return bLen;
-	}
-
-	if (bLen === 0) {
```

Comment:
Yes, here I guess that since we swapped the strings, we can remove this condition since if both strings have 0 length, condition in line 7 will handle it.